src/fable-library/Fable.Library.fsproj

```diff
@@ -1,30 +1,23 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <DefineConstants>$(DefineConstants);FABLE_COMPILER</DefineConstants>
-    <DefineConstants>$(DefineConstants);FX_NO_BIGINT</DefineConstants>
```

Comment:
We should keep this

Reply:
might have overseen that one...
VSCode messed up all the project files and I didn't notice right away ;-)